kDrive/Utils/UINavigationItem+Extension.swift

```diff
@@ -0,0 +1,29 @@
+/*
+ Infomaniak kDrive - iOS App
+ Copyright (C) 2021 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import UIKit
+
+extension UINavigationItem {
+    func dontShowBackButtonText() {
```

Comment:
Rename "dontShow" to "hide":
- It improves code readability 
- It looks more like Apple's own style (eg. view.isHidden)